dotCMS/src/main/java/com/dotmarketing/db/DbType.java

```diff
@@ -0,0 +1,46 @@
+package com.dotmarketing.db;
+
+/**
+ * Enum for the Database Types available
+ * @author Andre Curione
+ */
+public enum DbType {
+
+    MYSQL("MySQL"),
+    POSTGRESQL("PostgreSQL"),
+    ORACLE("Oracle"),
+    MSSQL("Microsoft SQL Server"),
+    H2("H2");
+
+    private String type;
+
+    DbType(final String type) {
+        this.type = type;
+    }
+
+    public String getDbType() {
+        return type;
+    }
+
+    public static DbType getDbType(final String dbType) {
+        DbType type = null;
+        switch (dbType) {
+            case "PostgreSQL":
+                type = DbType.POSTGRESQL;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "type". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)